kbase-extension/static/kbase/js/common/cellComponents/tabs/infoTab.js

```diff
@@ -119,7 +127,75 @@ define(['common/format', 'common/html', 'util/string'], (format, html, string) =
                 );
             }
 
-            const infoContainer = div(
+            /**
+             * A self-executing function which, given a string which purports to be an app description, is
+             * sanitized and repaired, if necessary.
+             *
+             * @param {string} descriptionText The description value for an app.
+             *
+             * @returns {string} The sanitized and possibly fixed description
+             */
+            const description = ((descriptionText) => {
+                if (!descriptionText) {
+                    return 'No description specified';
+                }
+
+                /**
+                 * "Fixes" the given node by recursively visiting it and all its children and applying fixes.
+                 *
+                 * At present, the fixes include:
+                 * - for the app info description:
+                 *  - replace the old data upload/download guide link with the current one
+                 *  - ensure that all KBase doc links open in a new window/tab
+                 *
+                 * @param {Node} node A DOM node
+                 * @returns {string} the input node, with fixes applied.
+                 */
+                const fix = (node) => {
```

Comment:
I think this is fine for now to patch over the bug, but it's really a band-aid over the real problem. The actual problem is that our docs URLs should be expected to be stable or have a redirect, and that one apparently does not. I mentioned in the ticket that fixing it here is fine, and those older narratives that use this would benefit from the update. However, these should probably not be hardcoded in the apps or in the Narrative, but configured somewhere. 

Not a concern right now, just noting my opinion.